### PR TITLE
Feature: Rubocop check if low queue is specify

### PR DESCRIPTION
**Pull request by hugobarthelemy**
**Rubocop: Low queue check** 👮‍♀️
Add Rubocop check if low queue is specify to Sidekiq call in migration.

### CODE REVIEW

**Comment by hugobarthelemy** [comment]
[(3) [CLOSURE] Forcer queue low dans les migration](https://trello.com/c/xjvfHbnv/169-3-closure-forcer-queue-low-dans-les-migration)

**Comment by hugobarthelemy** [comment]
[(3) [CLOSURE] Forcer queue low dans les migrations](https://trello.com/c/CRxWne2q/188-3-closure-forcer-queue-low-dans-les-migrations)

File: rubocop-klaxit/lib/gem_version.rb [hunk 1]
```diff
@@ -0,0 +1,3 @@
+module Klaxit
```
Comment (ThomasKlaxit) [inline rubocop-klaxit/lib/gem_version.rb]: Peut-être avoir `# frozen_string_literal: true` et retirer le `.freeze` ?

File: rubocop-klaxit/cops/specify_low_queue_in_migration.rb [hunk 2]
```diff
@@ -0,0 +1,73 @@
...
+  module Cop
+    # Cop that makes sure low queue is specify in worker call.
+    class SpecifyLowQueueInMigration < RuboCop::Cop::Cop
+      MSG = "Specify low queue when you call worker in migration with " \
+            "'HardWorker.set(queue: :app_name_low).perform_async(args)' " \
+            "or MyClass.delay(queue: 'app_name_low').some_method(1, 2, 3)"
+
+      def on_send(node)
+        return nil unless in_migration?(node)
+        #  || sidekiq_push?(node)
```
Comment (ThomasKlaxit) [inline rubocop-klaxit/cops/specify_low_queue_in_migration.rb]: Code mort ?

File: rubocop-klaxit/cops/specify_low_queue_in_migration.rb [hunk 3]
```diff
@@ -0,0 +1,73 @@
...
+      def low_queue_specify?(node)
+        low_queue_specify_for_perform?(node) ||
+          low_queue_specify_for_delay?(node) ||
+          low_queue_specify_for_push?(node)
+      end
+
+      def in_migration?(node)
+        dirname = File.dirname(node.location.expression.source_buffer.name)
+        dirname.end_with?("db/migrate", "db/geo/migrate") ||
+          in_post_deployment_migration?(node)
```
Comment (ThomasKlaxit) [inline rubocop-klaxit/cops/specify_low_queue_in_migration.rb]: Il manque la définition de cette fonction

**Comment by hugobarthelemy** [comment]
@BuonOmo Danger raise automatically when new Rubocop exception raise, so it's not necessary to add directly Danger.

**Comment by BuonOmo** [comment]
@hugobarthelemy that is not the point. I thought we would have only on gem for both our danger bot and our klaxit tools. Something like

```ruby
# Gemfile
gem "klaxit", require: false

# .rubocop.yml
require:
  - klaxit/rubocop

# Dangerfile
require "klaxit/danger"
klaxit.all
```

**Comment by hugobarthelemy** [comment]
@BuonOmo for the moment, for simplicity we prefer to leave it separate.

**Comment by hugobarthelemy** [comment]
[(2) [DANGER] Prise en compte des retours](https://trello.com/c/DnVcxA7b/225-2-danger-prise-en-compte-des-retours)

**Pull request merged**